tests/test_process_executor_spawn.py

```diff
@@ -5,7 +5,8 @@
 from ._executor_mixin import ExecutorMixin
 
 
-if sys.version_info[:2] > (3, 3):
+if (sys.version_info[:2] > (3, 3)
+        and not hasattr(sys, "pypy_version_info")):
```

Comment:
The spawn backend is also broken with `pypy`?

Reply:
Not necessarily completely broken, it might just be some timeouts, but the tests are already much too slow when running pypy so I think it's better to skip those.

PyPy has a much larger startup / warmup time compared to CPython. Therefore the loky test suite where we keep killing and recreating PyPy processes are significantly slower than the same tests run with CPython.

Reply:
FYI: the PyPy tests take 17 min (skipping forkserver and spawn) vs 6-7 min for the others.

I will also skip Fork to make them run faster.